### PR TITLE
[stable/nginx-ingress] Remove stats exporter

### PR DESCRIPTION
**Pull request by unguiculus**
A Prometheus metrics parser was added to nginx-ingress in 0.9-beta.3.
The separate stats exporter is no longer needed.

### CODE REVIEW

File: stable/nginx-ingress/values.yaml [hunk 1]
```diff
@@ -193,6 +193,27 @@ controller:
...
       type: ClusterIP
 
+  ## If controller.stats.enabled = true and controller.metrics.enabled = true, Prometheus metrics will be exported
+  ##
+  metrics:
+    enabled: false
+
+    service:
+      annotations: {}
+      # prometheus.io/scrape: true
```
Comment (lachie83) [inline stable/nginx-ingress/values.yaml]: The value needs to be quoted if they are to work if the comment is removed. It also might be worth adding `prometheus.io/port: 10254` as the default is currently 8080

Comment (chancez) [reply stable/nginx-ingress/values.yaml]: Also, uncommenting alone won't be enough, you need to indent the key, and remove the `{}` from the annotations key, I generally repeat the top-level key in the comments to make this easier:

```
      annotations: {}
      # annotations:
      #  "prometheus.io/scrape": true
```

**Review by lachie83 — CHANGES_REQUESTED** [review]
Overall looks good. Just a couple of nits.

File: stable/nginx-ingress/values.yaml (hunk 1 above)
Comment (unguiculus) [reply stable/nginx-ingress/values.yaml]: @lachie Good catch. Thanks.

@chancez Not sure about this. I'd rather not repeat the key. Also, this is not done anywhere else in the `values.yaml`.

**Review by lachie83 — APPROVED** [review]

**Pull request merged**